src/bin/mosaic-libraries.ts

```diff
@@ -6,6 +6,7 @@ import Logger from '../Logger';
 import OriginChainInteract from '../NewChain/OriginChainInteract';
 import MosaicConfig from '../Config/MosaicConfig';
 import PublishMosaicConfig from '../Config/PublishMosaicConfig';
+import { Utils } from '../Utils';
```

Comment:
Utils is exported as default. 
```suggestion
import Utils from '../Utils';
```